docs/.vuepress/config.js

```diff
@@ -1,78 +1,80 @@
-module.exports = {
-  dest: 'dist/docs',
-  title: 'chartjs-plugin-annotation',
-  description: 'Annotations for Chart.js',
-  theme: 'chartjs',
-  base: '/chartjs-plugin-annotation/',
-  head: [
-    ['link', {rel: 'icon', href: '/favicon.png'}],
-  ],
-  plugins: [
-    ['redirect', {
-      redirectors: [
-        // Default sample page when accessing /samples.
-        {base: '/samples', alternative: ['types/line']},
-      ],
-    }],
-  ],
-  themeConfig: {
-    repo: 'chartjs/chartjs-plugin-annotation',
-    logo: '/favicon.png',
-    lastUpdated: 'Last Updated',
-    searchPlaceholder: 'Search...',
-    editLinks: true,
-    docsDir: 'docs',
-    chart: {
-      imports: [
-        ['scripts/register.js'],
-        ['scripts/defaults.js'],
-        ['scripts/utils.js', 'Utils'],
-      ]
-    },
-    nav: [
-      {text: 'Home', link: '/'},
-      {text: 'Guide', link: '/guide/'},
-      {text: 'Samples', link: '/samples/'},
-    ],
-    sidebar: {
-      '/guide/': [
-        '',
-        'integration',
-        'usage',
-        'options',
-        'interaction',
-        {
-          title: 'Annotations',
-          collapsable: false,
-          children: [
-            'types/box',
-            'types/ellipse',
-            'types/line',
-            'types/point'
-          ]
-        }
-      ],
-      '/samples/': [
-        'intro',
-        {
-          title: 'Types',
-          collapsable: false,
-          children: [
-            'types/box',
-            'types/ellipse',
-            'types/line',
-            'types/point'
-          ],
-        },
-        {
-          title: 'Charts',
-          collapsable: false,
-          children: [
-            'charts/bar',
-            'charts/line',
-          ],
-        },
-      ]
-    }
-  }
-};
+module.exports = {

```

Comment:
Not quite sure what the whitespace change is in this file?

Reply:
no.. :( I'm gonna remove it

Reply:
Honestly, I was thinking it was related to last row (empty) but it's not... I don't why it's still there.

Reply:
line endigs?

Reply:
Tomorrow I’ll check

Reply:
Fixed